VisualizationBase/src/nodes/ViewItemNode.h

```diff
@@ -58,7 +62,13 @@ class VISUALIZATIONBASE_API ViewItemNode : public Super<UINode>
 		void setSpacingParent(ViewItemNode* spacingParent);
 		ViewItemNode* spacingParent() const;
 
+		virtual QJsonValue toJson() const;
+
 	private:
+		ViewItemNode(const ViewItem* parent);
+
+		const ViewItem* parent_{};
```

Comment:
Although, thinking of it, it might make some things a bit more troublesome as well. The VVIewItemNode would then probably directly receive the node it's meant to visualize, which would e.g. make it a bit more cumbersome to have it visualize both a node, and an empty spacing item.